interop/xds/server/server.go

```diff
@@ -53,6 +58,16 @@ var (
 	logger = grpclog.Component("interop")
 )
 
+const (
+	rpcBehaviorMDKey             = "rpc-behavior"
+	grpcPreviousRPCAttemptsMDKey = "grpc-previous-rpc-attempts"
+	sleepPfx                     = "sleep-"
+	keepOpenVal                  = "keep-open"
+	errorCodePfx                 = "error-code-"
+	successOnRetryPfx            = "succeed-on-retry-attempt-"
```

Comment:
Rename the const `succeedOnRetryPfx` as well?

Reply:
thanks had missed this